code/game/machinery/vending/marine_vending.dm

```diff
@@ -599,6 +600,7 @@
 					/obj/item/clothing/suit/storage/marine = 20,
 					/obj/item/clothing/head/helmet/marine = 20,
 					/obj/item/clothing/mask/rebreather/scarf = 10,
+					/obj/item/clothing/mask/rebreather = 10
```

Comment:
Check `cm_initialize.dm`, look for `/obj/item/clothing/mask/rebreather/scarf = round(scale * 30)`
Items added to these vendors must scale with population.